### PR TITLE
Ui test configs

### PR DESCRIPTION
**Pull request by amulware**
@willsb Let me know if I did ok. :D

screen shot 2018-03-22 at 17 46 04

🦑

### CODE REVIEW

File: Toggl.Giskard/Toggl.Giskard.csproj [hunk 1]
```diff
@@ -61,6 +61,19 @@
...
     <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release.UITest|AnyCPU' ">
+    <DebugType></DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Release.UITest</OutputPath>
+    <WarningLevel>4</WarningLevel>
+    <AndroidLinkSkip></AndroidLinkSkip>
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
+    <AndroidUseSharedRuntime>true</AndroidUseSharedRuntime>
```
Comment (willsb) [inline Toggl.Giskard/Toggl.Giskard.csproj]: No shared runtime, pls

File: build.cake [hunk 2]
```diff
@@ -353,8 +353,8 @@ Task("Build.Tests.Integration")
 
 Task("Build.Tests.UI")
     .IsDependentOn("Nuget")
-    .Does(BuildSolution("Debug"))
-    .Does(GenerateApk("Debug"));
+    .Does(BuildSolution("UITests"))
```
Comment (willsb) [inline build.cake]: Shouldn't this be `Release.UITests`?

Comment (amulware) [reply build.cake]: Don't think so. This is the name of the config in the solution.

File: Toggl.Giskard/Toggl.Giskard.csproj (hunk 1 above)
Comment (amulware) [reply Toggl.Giskard/Toggl.Giskard.csproj]: Squash and change yourself or wait patiently until tomorrow. :)

**Pull request merged**